features/facebookImport/src/model/storage.js

```diff
@@ -26,7 +26,7 @@ export default class Storage {
 
     async addFile() {
         return new Promise((resolve) => {
-            // File is already added by importFile, just refresh
+            // File is already added by refreshFiles, just refresh
```

Comment:
If the file is added by `refreshFiles`, why call it again? How I understand things the file is indeed added by (polyNav.)importFile

Reply:
It's called from here:
https://github.com/polypoly-eu/polyPod/blob/30641a3cca771f6269004f4a8ee999dab1b78fe4/features/facebookImport/src/components/facebook-import.js#L45-L47
However, it returns a `Promise` and we're not awaiting for it.
I really don't know what's going on here. I've adjusted the PR body to reflect my impression.

Reply:
`Storage` is used here:

https://github.com/polypoly-eu/polyPod/blob/67d136ba6e3bc552897e82e05e87dc6a39bd09ec/features/facebookImport/src/context/importer-context.jsx#L175-L185

and `refreshFiles` is called repeatedly, but that function reads files using :exploding_head: `const files = await polyOut.readdir("");` This does not even include any kind of state information from the pod. Again, I'm tempted to just eliminate all calls to this to see what happens.
This comes from trying to create unit tests for `Storage`, actually... #175